schema/deploy/shipping/views.sql

```diff
@@ -51,6 +51,8 @@ drop view if exists shipping.incidence_model_observation_v3;
 drop view if exists shipping.incidence_model_observation_v2;
 drop view if exists shipping.incidence_model_observation_v1;
 
+drop view if exists shipping.linelist_data_for_wa_doh_v1;
```

Comment:
This is a correction to the DOH view that's already in master. The drop needs to be before shipping.hcov19_presence_absence_result_v1.